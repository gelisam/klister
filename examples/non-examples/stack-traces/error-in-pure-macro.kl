#lang "prelude.kl"

(example (pure (error 'surprise-error)))