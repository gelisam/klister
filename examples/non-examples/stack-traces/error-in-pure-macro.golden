In Phase: p0:
    error-in-pure-macro.kl:3.24-3.38: surprise-error
    stack trace:
    ---- in pure macro
    ---- Halt
